### PR TITLE
✨ add the core class

### PR DESCRIPTION
**Pull request by Quentintnrl**
I can't make unit test for the core because it's only call graphic and game method so we will need to make manual test

### CODE REVIEW

**Review by 6im0n — APPROVED** [review]
Good for me !

File: core/Core.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    delete _graphic;
+    delete _game;
+    delete _menu;
+    delete _graphicLoader;
+    delete _gameLoader;
+}
+
+void arcade::Core::run()
+{
+    while (5) {
```
Comment (Njord201) [inline core/Core.cpp:31]: Pourquoi while(5) ?

**Review by Njord201 — CHANGES_REQUESTED** [review]

File: core/Core.cpp (hunk 1 above)
Comment (Quentintnrl) [reply core/Core.cpp:31]: Parce qu'il faut bien que le pogramme boucle

**Review by Njord201 — APPROVED** [review]
:+1:

**Pull request merged**